extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerService.ts

```diff
@@ -6,16 +6,18 @@
 
 import { ResourceServiceBase, GraphData } from '../resourceTreeDataProviderBase';
 import { azureResource } from 'azurecore';
+import { serversQuery } from './serverQueryStrings';
 
 export interface DbServerGraphData extends GraphData {
 	properties: {
 		fullyQualifiedDomainName: string;
 		administratorLogin: string;
+		connectivityEndpoints?: { sql: string };
+		managedResourceGroupName?: string;
+		sqlAdministratorLogin?: string;
```

Comment:
As mentioned below - these aren't valid properties for normal servers so they should go into their own type.